docs/user-guide/src/capm3/pivoting.md

```diff
@@ -0,0 +1,167 @@
+  `clusterctl` essentially adds the following labels in the CRDs of each related object.
+
+  ```yaml
+  labels:
+  - clusterctl.cluster.x-k8s.io: ""
+  - cluster.x-k8s.io/provider: "<provider-name>"
+  ```
+
+  So if the clusters are not initialized using `clusterctl`, all the CRDS of the
+  objects to be moved to target cluster needs to have these labels both in
+  bootstrap cluster and target cluster before performing the move.
+
+  **Note**: This is not recommended, since
+  the way `clusterctl` identifies objects to manage might change in the future, so
+  it's always safe to install CRDs and controllers through the `clusterctl init` sub-command.
+
+* **BMH objects have correct status annotation.**
+
+  Since BMH _status_ holds important information regarding the BMH itself, BMH with
+  status has to be moved and BMH has to be reconstructed with correct status in
+  target  cluster before it is being reconciled. This is now done through BMH
+  status annotation in BMO.
+
+* **Maintain connectivity towards provisioning network.**
+
+  Baremetal machines boot over a network with a DHCP server. This requires
+  maintaining a fixed IP end points towards the provisioning network. This is
+  achieved through _keepalived_. A new container is added namely
+  _ironic-endpoint-keepalived_ which maintains the Ironic Endpoint using
+  keepalived. The motivation behind maintaining Ironic Endpoint with Keepalived
+  is to ensure that the Ironic Endpoint IP is also passed onto the target
+  cluster control plane. This also guarantees that once moving is done and the
+   management cluster is taken down, target cluster controlplane can re-claim
+   the Ironic endpoint IP through keepalived. The end goal is to make Ironic
+   endpoint reachable in the target cluster.
+
+* **BMO is deployed as part of CAPM3.**
+
+  If not, it has to be deployed before the `clusterctl init` and the BMH CRDs
+  need to be labeled accordingly manually. This is a prerequisite for both the
+  management and the target cluster.
+
+* **Objects should have a proper owner reference chain.**
+
+  `clusterctl move` moves all the objects to the target cluster following the
+[owner reference chain](https://cluster-api.sigs.k8s.io/clusterctl/provider-contract.html#ownerreferences-chain).
+So, it is necessary to verify that all the desired objects that needs to
+be moved to the target cluster have a proper owner reference chain.
+
+## Step by step pivoting process
+
+As described in
+[clusterctl](https://cluster-api.sigs.k8s.io/clusterctl/commands/move.html)
+the whole process of bootstrapping a management cluster to moving
+objects to target cluster can be described as follows:
+
+The move process can be bounded with the creation of a temporary bootstrap
+cluster used to provision a target Management cluster.
+
+This can now be achieved with the following procedure:
+
+1. Create a temporary bootstrap cluster, e.g. using Kind or Minikube using
+ `clusterctl`.
```

Comment:
kind or minikube is not setup using clusterctl. Rather the provider components in kind or minikube are. This line needs to be tweaked a little bit to phrase this appropriately.